azure-pipelines.yml

```diff
@@ -201,6 +201,17 @@ jobs:
       buildType: 'current'
       downloadType: 'specific'
       downloadPath: '$(Build.SourcesDirectory)'
+  - task: ManifestGeneratorTask@0
+    displayName: 'SBOM Generation Task'
+    inputs:
+      BuildDropPath: '$(Build.SourcesDirectory)'
+      BuildComponentPath: '$(Build.SourcesDirectory)'
+      Verbosity: 'Verbose'
```

Comment:
Why verbose? I think other languages workers are keeping it as 'Information'.
https://github.com/Azure/azure-functions-java-worker/pull/494/files

Reply:
yes java chose information, but I chose it because I prefer more logging and it doesn't hurt.